### ISSUE TITLE
Sea Dungeon (Gallen's graveyard) Bugs, Please fix this...

### ISSUE THREAD

**Issue by JeffTheNub** [Wynncraft/Issues#7184]
To be honest here, I just continued playing Wynncraft since 3 days ago and I found it added new dungeons, but the worst thing is there are a lot of bugs there. First, if you were doing this dungeon with a friend (or two), after you killed the Ancient Pirate, you should go into a cutscene. But only if we didn't went in the door to activate the cutscene at the same time. We'll be stuck between the gate and a wall of barrier! 
This is not main point, I could try to leap through the barrier with my spells. BUT!!! Later on, the Pirate crews weren't talking anything and they didn't fire at us! That means we were STUCK in the dungeon. Don't say we can do /class or change into another world to solve the problem, actually, it makes it even worse, when I did /class or change a world, I was stuck in the Command Blocks and redstone wirings , but my friend said he saw me hovering on the sky at that that moment. I asked a few mods and they said they have no clue how this could happen. 

In Conclusion,  Please fix the bugs that I said above please! That'll be muchly appreciated.

**Comment by thecoolkogut4** [comment]
I'm pretty sure the dungeons were made to be done all at once, and to not wait X minutes/hours between each stage.

**Comment by kkbirdy** [comment]
/kill should still work to get you out, have you tried that?

**Comment by AscendedKitten** [comment]
I am sorry for the redstone/commandblock issues you experienced, but unfortunately the only way to leave is using a teleport scroll or /kill for now.
